modules/ROOT/pages/guided-setup.adoc

```diff
@@ -25,12 +25,54 @@ This may be the US control plane, the EU control plane, the MuleSoft Government
 +
 If your Anypoint account has the organization administrator role, the guided setup tool can create the connected app for you.
 +
-Otherwise, create the connected app or ask an organization administrator to create it, then provide the app details so the guided setup can perform the authentication and authorization.
+Otherwise, create the connected app or ask an organization administrator to create it with these steps:
```

Comment:
May be we can use this section:
https://docs.mulesoft.com/api-community-manager/manual-setup#create-a-key-pair